tests/jsx-multiline-assign/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -22,22 +22,12 @@ const comp4 = <div style={styles} key="something">Create wrapping parens and ind
 const comp5 = <div>Keep it on one line.</div>;
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 const comp1 = (
-  <div style={styles} key="something">
-    Keep the wrapping parens.
-  </div>
+  <div style={styles} key="something">Keep the wrapping parens.</div>
```

Comment:
I still think this was more readable before, but I have no idea which heuristic would make it better...